packages/google_maps_flutter/ios/Classes/GoogleMapController.h

```diff
@@ -5,10 +5,12 @@
 #import <Flutter/Flutter.h>
 #import <GoogleMaps/GoogleMaps.h>
 #import "GoogleMapMarkerController.h"
+#import "GoogleMapPolylineController.h"
 
 // Defines map UI options writable from Flutter.
 @protocol FLTGoogleMapOptionsSink
-- (void)setCameraTargetBounds:(GMSCoordinateBounds*)bounds;
+- (void)setCamera:(GMSCameraPosition *)camera;
```

Comment:
Is there a reason this method is added?

Reply:
@awazgyawali looks like you added this one. Could you take a look at this?

Reply:
I don't recall it either, wasn't it already on the master branch. I will check if this function is used and remove it.

Reply:
Removed.